apps/src/codebridge/FileBrowser/index.tsx

```diff
@@ -273,14 +274,27 @@ const InnerFileBrowser = React.memo(
                       iconStyle={'solid'}
                       className={moduleStyles.rowIcon}
                     />
-                    <span
-                      className={classNames(moduleStyles.nameContainer, {
-                        [moduleStyles.acceptingDrop]:
-                          f.id === dropData?.id && dragData?.parentId !== f.id,
-                      })}
+
+                    <OverflowTooltip
+                      tooltipProps={{
+                        text: f.name,
+                        tooltipId: `folder-tooltip-${f.id}`,
+                        size: 's',
+                        direction: 'onBottom',
+                      }}
+                      tooltipOverlayClassName={moduleStyles.nameContainer}
+                      className={moduleStyles.nameContainer}
                     >
-                      {f.name}
-                    </span>
+                      <span
+                        className={classNames({
+                          [moduleStyles.acceptingDrop]:
+                            f.id === dropData?.id &&
+                            dragData?.parentId !== f.id,
```

Comment:
This predates this PR, so take or leave here, but I wonder if adding a comment to explain this logic is worthwhile.

It looks like this is used to apply a special style when the element is the intended drop target (f.id === dropData?.id) and the element is not the parent of the dragged item (dragData?.parentId !== f.id) (which would prevent an item from being dropped onto its parent?).

Reply:
@thomasoniii added this